src/test/kotlin/CalculateTest.kt

```diff
@@ -0,0 +1,11 @@
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class SumTest {
+  @Test
+  fun testCalculate() {
```

Comment:
Nit: Usually we would name a test like e.g. `returns correct calculation` or `should calculate sum correctly` 

Example: https://github.com/squareup/cash-server/blob/aee3044ad140239debc618e63393e7e7bc493c73/topographer/service/src/test/kotlin/com/squareup/cash/topographer/service/persistence/ArtifactNameManagerTest.kt#L101